website/docs/r/virtual_machine.html.markdown

```diff
@@ -117,6 +117,8 @@ The following arguments are supported:
 * `listen` - (Required) Binding address.
 * `port` - (Optional) Binding Port.
 * `keymap` - (Optional) Keyboard mapping.
+* `passwd` - (Optional) VNC's password.
+* `random_passwd` - (Optional) Randomized VNC's password.
```

Comment:
Conflicts should be mentioned on the documentation

Reply:
done